Resources/views/app/base.html.twig

```diff
@@ -18,6 +18,8 @@
     {{ include('partials/navbar_user.html.twig') }}
 {% endblock %}
 
+{% block title %}AAK-tid – {{ block('page_title') }}{% endblock %}
```

Comment:
If you just want to add a prefix to whatever the original `title` block renders, you can use https://twig.symfony.com/doc/3.x/functions/parent.html here:

```suggestion
{% block title %}AAK-tid – {{ parent() }}{% endblock %}
```